src/utils/scenarioParameters/ScenarioParameterInputFactory.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Cosmo Tech.
```

Comment:
Maybe rename folder with 'factory' in names at least (same thing for 'inputComponent')